org.eclipse.m2e.core.tests/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,6 @@ Require-Bundle: org.eclipse.m2e.tests.common,
  org.eclipse.core.runtime,
  org.eclipse.m2e.maven.runtime
 Import-Package: javax.annotation;version="1.2.0",
- org.apache.commons.io;version="2.11.0"
+ org.apache.commons.io
```

Comment:
@laeubi any idea, why Tycho fails to resolver this package import when it has a package version and the exporting bundle is from a p2-repo (looks like a original bundle), but succeeds with version if the providing bundle is from a Maven-target?